src/main/java/com/esri/samples/featurelayers/time_based_query/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Time Based Query</h1>
+
+<p>Demonstrates how to query a feature table to only show features in a specific time extent.</p>
+
+<p><img src="TimeBasedQuery.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To query features by a time extent:</p>
+
+<ol>
+    <li>Create a <code>ServiceFeatureTable</code> from the URL of a feature service.</li>
+    <li>After loading the service feature table, create <code>QueryParameters</code>.</li>
+    <li>Create two <code>Calendar</code> objects with the beginning and ending timestamps and create a 
+    <code>TimeExtent</code> with them.</li>
+    <li>Set the time extent with <code>queryParameters.setTimeExtent(timeExtent)</code></li>
+    <li>Populate the table with features in the time extent with <code>featureTable.popuateFromServiceAsync(queryParameters, true, outputFields)</code>.
```

Comment:
Need to add a step for setting to feature request mode to manual or calling `popuateFromServiceAsync` will throw an error.

Reply:
I didn't know that. Thanks!